.prettierrc.json

```diff
@@ -1,5 +1,5 @@
 {
-    "tabSize": 2,
```

Comment:
I think `tabWidth` is the correct option: https://prettier.io/docs/en/options.html#tab-width